### DISCUSSION TITLE
用户码

### DISCUSSION THREAD

**Discussion by yihangyingzi** [share121/inter-knot#5172] [其他]
安卓app为啥老弹用户码出来，不授权设备还不让用？<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" >

**Comment by Junjun280** [comment]
正常来说，你登录过了就不会弹用户码来着，这应该是bug，可以去反馈

**Comment by Ag17-CorinWickes** [comment]
应该是网络变动导致的

**Comment by Yizakl** [comment]
可以在设置里面切换公共接口哦喵

**Comment by HKLHaoBin** [comment]
用户可以通过公共的github API获取绳网讨论，也可以通过个人的github API获取绳网，但是公共的github API只有一个，访问的人一旦多起来，就容易超限。此时可以通过个人的github API获取绳网。

**Reply by Yizakl** [reply]
# 大佬！
